server/graphql/v1/types.js

```diff
@@ -829,18 +817,6 @@ export const ExpenseType = new GraphQLObjectType({
           }
         },
       },
-      canSeePrivateInfo: {
```

Comment:
This one wasn't used for API V1, so it's easier to delete with together with `ExpenseViewPermissions`